tests/bootstrap.php

```diff
@@ -19,65 +19,101 @@
-		require dirname( __DIR__ ) . '/woocommerce-gutenberg-products-block.php';
+/**
+ * Adds WooCommerce testing framework classes.
+ */
+function wc_test_includes() {
+	$wc_tests_framework_base_dir = wc_dir() . '/tests';
+	// WooCommerce test classes.
+	// Framework.
+	require_once $wc_tests_framework_base_dir . '/framework/class-wc-unit-test-factory.php';
+	require_once $wc_tests_framework_base_dir . '/framework/class-wc-mock-session-handler.php';
+	require_once $wc_tests_framework_base_dir . '/framework/class-wc-mock-wc-data.php';
+	require_once $wc_tests_framework_base_dir . '/framework/class-wc-mock-wc-object-query.php';
+	require_once $wc_tests_framework_base_dir . '/framework/class-wc-mock-payment-gateway.php';
+	require_once $wc_tests_framework_base_dir . '/framework/class-wc-payment-token-stub.php';
+	require_once $wc_tests_framework_base_dir . '/framework/vendor/class-wp-test-spy-rest-server.php';
+	// Test cases.
+	require_once $wc_tests_framework_base_dir . '/includes/wp-http-testcase.php';
+	require_once $wc_tests_framework_base_dir . '/framework/class-wc-unit-test-case.php';
+	require_once $wc_tests_framework_base_dir . '/framework/class-wc-api-unit-test-case.php';
+	require_once $wc_tests_framework_base_dir . '/framework/class-wc-rest-unit-test-case.php';
+	// Helpers.
+	require_once $wc_tests_framework_base_dir . '/framework/helpers/class-wc-helper-product.php';
+	require_once $wc_tests_framework_base_dir . '/framework/helpers/class-wc-helper-coupon.php';
+	require_once $wc_tests_framework_base_dir . '/framework/helpers/class-wc-helper-fee.php';
+	require_once $wc_tests_framework_base_dir . '/framework/helpers/class-wc-helper-shipping.php';
+	require_once $wc_tests_framework_base_dir . '/framework/helpers/class-wc-helper-customer.php';
+	require_once $wc_tests_framework_base_dir . '/framework/helpers/class-wc-helper-order.php';
+	require_once $wc_tests_framework_base_dir . '/framework/helpers/class-wc-helper-shipping-zones.php';
+	require_once $wc_tests_framework_base_dir . '/framework/helpers/class-wc-helper-payment-token.php';
+	require_once $wc_tests_framework_base_dir . '/framework/helpers/class-wc-helper-settings.php';
+}
+
+function wc_load_core() {
+	define( 'WC_TAX_ROUNDING_MODE', 'auto' );
+	define( 'WC_USE_TRANSACTIONS', false );
+	echo esc_html( 'Loading WooCommerce plugin' . PHP_EOL );
+	require_once wc_dir() . '/woocommerce.php';
+}
+
+function wc_install_core() {
+	// Clean existing install first.
+	define( 'WP_UNINSTALL_PLUGIN', true );
+	define( 'WC_REMOVE_ALL_DATA', true );
+	include wc_dir() . '/uninstall.php';
+	$GLOBALS['wp_roles'] = null; // WPCS: override ok.
+	wp_roles();
+	echo esc_html( 'Installing WooCommerce...' . PHP_EOL );
+}
+
+/**
+ * Manually load the plugin being tested.
+ */
+tests_add_filter(
+	'muplugins_loaded',
+	function() {
+		wc_load_core();
+		// install blocks plugin
+		wc_blocks_install();
 	}
 );
 
+tests_add_filter('setup_theme','wc_install_core');
```

Comment:
Should this be padded with spaces?
```suggestion
tests_add_filter( 'setup_theme', 'wc_install_core' );
```

Reply:
fixed in f3049e0 and d2dee63